nupic/swarming/exp_generator/ExpGenerator.py

```diff
@@ -375,7 +375,8 @@ def _generateFileFromTemplates(templateFileNames, outputFilePath,
   """
 
   # Find out where we're running from so we know where to find templates
-  installPath = os.path.dirname(__file__)
+  installPath = os.path.dirname(__file__) #TODO: Will want to change the
```

Comment:
you may also want to get the `abspath` of `__file__` before getting the `dirname` to avoid relative path issues